Spring Security Taglib not found

When I try to run the example I get:

```
javax.faces.view.facelets.TagAttributeException: //C:/Users/simon/Workspace/jsf/joinfaces-primefaces-jooq/target/classes/META-INF/resources/starter.xhtml @32,79 rendered="#{sec:isFullyAuthenticated()}" Function [sec:isFullyAuthenticated] not found
	at com.sun.faces.facelets.tag.TagAttributeImpl.getValueExpression(TagAttributeImpl.java:388) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.TagAttributeImpl.getValueExpression(TagAttributeImpl.java:338) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentRule$ValueExpressionMetadata.applyMetadata(ComponentRule.java:68) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.MetadataImpl.applyMetadata(MetadataImpl.java:40) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.MetaTagHandler.setAttributes(MetaTagHandler.java:100) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.setAttributes(DelegatingMetaTagHandler.java:92) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.doNewComponentActions(ComponentTagHandlerDelegateImpl.java:453) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.apply(ComponentTagHandlerDelegateImpl.java:155) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.apply(DelegatingMetaTagHandler.java:111) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.CompositeFaceletHandler.apply(CompositeFaceletHandler.java:55) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.ui.DefineHandler.applyDefinition(DefineHandler.java:65) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.ui.CompositionHandler.apply(CompositionHandler.java:166) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFaceletContext$TemplateManager.apply(DefaultFaceletContext.java:372) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFaceletContext.includeDefinition(DefaultFaceletContext.java:342) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.ui.InsertHandler.apply(InsertHandler.java:70) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.CompositeFaceletHandler.apply(CompositeFaceletHandler.java:55) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.applyNextHandler(DelegatingMetaTagHandler.java:137) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.apply(ComponentTagHandlerDelegateImpl.java:179) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.apply(DelegatingMetaTagHandler.java:111) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.applyNextHandler(DelegatingMetaTagHandler.java:137) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.apply(ComponentTagHandlerDelegateImpl.java:179) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.apply(DelegatingMetaTagHandler.java:111) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.applyNextHandler(DelegatingMetaTagHandler.java:137) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.apply(ComponentTagHandlerDelegateImpl.java:179) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.apply(DelegatingMetaTagHandler.java:111) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.applyNextHandler(DelegatingMetaTagHandler.java:137) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.jsf.ComponentTagHandlerDelegateImpl.apply(ComponentTagHandlerDelegateImpl.java:179) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.DelegatingMetaTagHandler.apply(DelegatingMetaTagHandler.java:111) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.view.facelets.CompositeFaceletHandler.apply(CompositeFaceletHandler.java:55) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.compiler.NamespaceHandler.apply(NamespaceHandler.java:52) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.compiler.EncodingHandler.apply(EncodingHandler.java:46) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFacelet.include(DefaultFacelet.java:271) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFacelet.include(DefaultFacelet.java:330) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFacelet.include(DefaultFacelet.java:309) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFaceletContext.includeFacelet(DefaultFaceletContext.java:162) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.tag.ui.CompositionHandler.apply(CompositionHandler.java:133) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.compiler.NamespaceHandler.apply(NamespaceHandler.java:52) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.compiler.EncodingHandler.apply(EncodingHandler.java:46) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.facelets.impl.DefaultFacelet.apply(DefaultFacelet.java:120) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.application.view.FaceletViewHandlingStrategy.buildView(FaceletViewHandlingStrategy.java:358) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.lifecycle.RenderResponsePhase.execute(RenderResponsePhase.java:78) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.lifecycle.Phase.doPhase(Phase.java:76) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at com.sun.faces.lifecycle.LifecycleImpl.render(LifecycleImpl.java:199) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.webapp.FacesServlet.executeLifecyle(FacesServlet.java:708) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at javax.faces.webapp.FacesServlet.service(FacesServlet.java:451) ~[jakarta.faces-2.3.14.jar:2.3.14]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-embed-websocket-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:320) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:126) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:90) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:118) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:137) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:111) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:158) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:200) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:116) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:92) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:77) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:105) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:215) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:178) ~[spring-security-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:358) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:271) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.1.RELEASE.jar:5.2.1.RELEASE]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:526) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:861) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1579) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-embed-core-9.0.27.jar:9.0.27]
	at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
Caused by: javax.el.ELException: Function [sec:isFullyAuthenticated] not found
	at org.apache.el.lang.ExpressionBuilder.visit(ExpressionBuilder.java:219) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at org.apache.el.parser.SimpleNode.accept(SimpleNode.java:142) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at org.apache.el.lang.ExpressionBuilder.prepare(ExpressionBuilder.java:161) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at org.apache.el.lang.ExpressionBuilder.build(ExpressionBuilder.java:179) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at org.apache.el.lang.ExpressionBuilder.createValueExpression(ExpressionBuilder.java:242) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at org.apache.el.ExpressionFactoryImpl.createValueExpression(ExpressionFactoryImpl.java:67) ~[tomcat-embed-el-9.0.27.jar:9.0.27]
	at com.sun.faces.facelets.tag.TagAttributeImpl.getValueExpression(TagAttributeImpl.java:372) ~[jakarta.faces-2.3.14.jar:2.3.14]
	... 110 common frames omitted

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,16 +8,12 @@ cache:
   - $HOME/.m2
 addons:
   firefox: latest
+  chrome: stable
 
 before_script:
   - echo "Extracting firefox and setting PATH variable..."
   - tar -xjf /tmp/firefox-*.tar.bz2 --directory /tmp
   - export PATH="/tmp/firefox:$PATH"
-  - echo "Downloading chrome..."
-  - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-  - sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-  - sudo apt-get update
-  - sudo apt-get install google-chrome-stable
 
 install: ./mvnw -B clean validate
 

--- a/README.md
+++ b/README.md
@@ -33,10 +33,12 @@ mvn clean install
 
 3- Run
 ```Shell
-java -jar target/joinfaces-example-4.0.1-SNAPSHOT.jar
+java -jar target/joinfaces-example-4.1.1.jar
 ```
 
 4- Access starter page at **http://localhost:8080/**
+
+Optional: If your IDE is showing build errors install [Lombok](https://projectlombok.org/setup/overview)
 
 ## Key Files
 
@@ -46,7 +48,7 @@ Includes joinfaces starter dependency. All other jsf dependencies are included t
 
 ```xml
 <properties>
-   <joinfaces.version>4.0.1</joinfaces.version>
+   <joinfaces.version>4.1.1</joinfaces.version>
 </properties>
 
 <dependencyManagement>

--- a/pom.xml
+++ b/pom.xml
@@ -25,10 +25,10 @@
         <maven-model.version>3.6.0</maven-model.version>
 
         <!-- test library versions -->
-        <selenium.version>3.141.5</selenium.version>
-        <htmlunit-driver.version>2.33.2</htmlunit-driver.version>
-        <htmlunit.version>2.33</htmlunit.version>
-        <webdrivermanager.version>3.0.0</webdrivermanager.version>
+        <selenium.version>3.141.59</selenium.version>
+        <htmlunit-driver.version>2.37.0</htmlunit-driver.version>
+        <htmlunit.version>2.37.0</htmlunit.version>
+        <webdrivermanager.version>3.8.1</webdrivermanager.version>
         <jarchivelib.version>1.0.0</jarchivelib.version>
 
         <jacoco-maven-plugin.version>0.8.2</jacoco-maven-plugin.version>        
@@ -185,6 +185,12 @@
         <dependency>
             <groupId>commons-fileupload</groupId>
             <artifactId>commons-fileupload</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.projectlombok</groupId>

--- a/src/main/java/org/joinfaces/example/view/JoinFacesStarterService.java
+++ b/src/main/java/org/joinfaces/example/view/JoinFacesStarterService.java
@@ -176,7 +176,7 @@ public class JoinFacesStarterService {
 
 	private Model createModel() throws MalformedURLException, IOException, XmlPullParserException {
 		MavenXpp3Reader mavenXpp3Reader = new MavenXpp3Reader();
-		return mavenXpp3Reader.read(new URL("http://repo1.maven.org/maven2/org/joinfaces/joinfaces-dependencies/"
+		return mavenXpp3Reader.read(new URL("https://repo1.maven.org/maven2/org/joinfaces/joinfaces-dependencies/"
 				+ joinfacesVersion + "/joinfaces-dependencies-" + joinfacesVersion + ".pom").openStream());
 	}
 

--- a/src/main/resources/META-INF/resources/starter.xhtml
+++ b/src/main/resources/META-INF/resources/starter.xhtml
@@ -7,7 +7,7 @@
                 xmlns:p="http://primefaces.org/ui"
                 xmlns:pe="http://primefaces.org/ui/extensions"
                 xmlns:o="http://omnifaces.org/ui"
-                xmlns:sec="http://www.springframework.org/security/tags"
+                xmlns:sec="https://joinfaces.org/security"
                 template="/template.xhtml">
     <ui:define name="content" >
         <p:panelGrid columns="2" layout="grid">

--- a/src/test/java/org/joinfaces/example/view/AbstractPageIT.java
+++ b/src/test/java/org/joinfaces/example/view/AbstractPageIT.java
@@ -21,6 +21,7 @@ import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.firefox.FirefoxOptions;
 import org.openqa.selenium.firefox.FirefoxProfile;
@@ -66,7 +67,10 @@ public class AbstractPageIT {
 
 	private static WebDriver getChromeDriver() {
 		WebDriverManager.getInstance(ChromeDriver.class).setup();
-		return new ChromeDriver();
+		ChromeOptions options = new ChromeOptions();
+		options.addArguments("--headless");
+		options.addArguments("--disable-gpu");
+		return new ChromeDriver(options);
 	}
 
 	private static WebDriver getFirefoxDriver() {

